applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -259,3 +273,162 @@ def _SetNodalProperties(self):
         # Transfer the obtained properties to the nodes
         KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.DENSITY, rho, self.main_model_part.Nodes)
         KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.VISCOSITY, kin_viscosity, self.main_model_part.Nodes)
+
+    # TODO: I THINK THIS SHOULD BE MOVED TO THE BASE PYTHON SOLVER
+    def is_restarted(self):
+        # this function avoids the long call to ProcessInfo and is also safer
+        # in case the detection of a restart is changed later
+        return self.main_model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED]
+
+    def get_estimate_dt_utility(self):
```

Comment:
I would take the chance and use the style-guide conforming names:
`_GetEstimateDtUtility`

Reply:
I thought the style guide is snake case :man_facepalming:

Reply:
https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#classes-1
;)